### DISCUSSION TITLE
Neovim nighlty 0.10.0 installation

### DISCUSSION THREAD

**Discussion by rwxmad** [neovim/neovim#25413] [Q&A] [answered]
how can I install neovim prerelease 0.10.0 using homebrew, or so that I don’t have to call new from the folder where I downloaded it on macos

**Comment by clason** [comment, marked as answer]
You can't; build from source or download from https://github.com/neovim/neovim/releases/tag/nightly.

(And with all due respect, if this is difficult for you, I recommend sticking with actual releases. There should be no reason to use nightlies for production use.)

**Reply by rwxmad** [reply]
It’s really not clear to me how to build and run the neovim from source code, but thank you for the answer!

**Reply by clason** [reply]
Then I really recommend just sticking to released versions! Nightlies routinely break, and you'll be expected to help investigate problems if you open an issue. (The information is in the Wiki under "build", if you are curious.)

**Comment by theherk** [comment]
I'm not about to disagree with @clason; you should indeed stick to the released version if troubleshooting isn't something you're interested in, or how to run it isn't quite clear already. But... nobody get's from not knowing to knowing without some guidance along the way. Therefore...

There are [installation instructions for MacOS](https://github.com/neovim/neovim/wiki/Installing-Neovim#macos--os-x). All you really have to do is copy and run the given command.

```sh
curl -LO https://github.com/neovim/neovim/releases/download/nightly/nvim-macos.tar.gz
tar xzf nvim-macos.tar.gz
./nvim-macos/bin/nvim
```

But you probably got that far as it seems your underlying question is how to run without running a special command referencing the location. There are a few methods, but the most simple by far is just to alias your command.

bash (assuming you were in your `HOME` when you ran that command):

    alias nvim='~/nvim-macos/bin/nvim'

fish:

    abbr -a nvim --position command ~/nvim-macos/bin/nvim

Then when you run `nvim` you get the latest version you've downloaded, and all you have to do is remove that alias to continue using the brew installed version. Or you can call the alias something like `nvim10` if you want to run either at anytime.
